enclave/src/cert.rs

```diff
@@ -275,21 +266,32 @@ pub fn verify_mra_cert(cert_der: &[u8]) -> Result<(), sgx_status_t> {
 		now_func.sgx_error()?,
 	) {
 		Ok(_) => info!("Cert is good"),
-		Err(e) => error!("Cert verification error {:?}", e),
+		Err(e) => {
+			error!("Cert verification error {:?}", e);
+			return Err(sgx_status_t::SGX_ERROR_UNEXPECTED)
+		},
```

Comment:
return an error when we run into this, instead of just logging the error and continuing